ro.fxdata.taskmonitor.viewer.appdata.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>ro.fxdata.taskmonitor.viewer</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+  <name>TkmViewer</name>
+  <summary>Visualize TaskMonitor SQLite3 output data</summary>
+  <description>
+    <p>
+      TkmViewer provides a good way to visiualize output data from TaskMonitor.
+    </p>
+    <p>Current features:</p>
+    <ul>
+      <li>Adaptive UI</li>
+      <li>System overview dashboard</li>
+      <li>Process and container data</li>
+      <li>Other system info data (buddyinfo, diskstats, etc)</li>
+    </ul>
+  </description>
+  <screenshots>
+    <screenshot>
+      <image>https://gitlab.com/taskmonitor/tkmviewer/-/raw/master/data/screenshots/tkmviewer-1.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://gitlab.com/taskmonitor/tkmviewer/-/raw/master/data/screenshots/tkmviewer-2.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://gitlab.com/taskmonitor/tkmviewer/-/raw/master/data/screenshots/tkmviewer-3.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://gitlab.com/taskmonitor/tkmviewer/-/raw/master/data/screenshots/tkmviewer-4.png</image>
+    </screenshot>
+  </screenshots>
+  <launchable type="desktop-id">ro.fxdata.taskmonitor.desktop</launchable>
+  <url type="homepage">https://gitlab.com/taskmonitor/tkmviewer</url>
+  <developer_name>Alin Popa</developer_name>
+  <update_contact>alin.popa_at_fxdata.ro</update_contact>
+  <requires>
+    <display_length compare="ge">640</display_length>
+  </requires>
+  <recommends>
+    <control>keyboard</control>
+    <control>pointing</control>
+    <control>touch</control>
+  </recommends>
+  <releases>
+    <release version="1.0.0" date="2022-08-17">
+      <description>
+        <p>Initial release</p>
+        <ul>
+          <li>Create initial application design</li>
+        </ul>
+      </description>
+    </release>
+  </releases>
+  <content_rating type="oars-1.1"/>
+  <provides>ro.fxdata.taskmonitor.desktop</provides>
```

Comment:
```suggestion
  <launchable type="desktop-id">ro.fxdata.taskmonitor.desktop</launchable>
```

Reply:
except that the provides was for a different desktop file. and that the suggestion doesn't match the app-id. Hence the build failure.